CMakeLists.txt

```diff
@@ -46,6 +46,10 @@ CHECK_INCLUDE_FILE("alloca.h" HAVE_ALLOCA_H)
 #CHECK_INCLUDE_FILE("getopt.h" HAVE_GETOPT_H)
 CHECK_INCLUDE_FILE("malloc.h" HAVE_MALLOC_H)
 CHECK_INCLUDE_FILE("pcre2.h" HAVE_PCRE2_H)
+include(CheckSymbolExists)
+check_symbol_exists("memmem" "string.h" HAVE_MEMMEM)
+check_symbol_exists("strcasestr" "string.h" HAVE_MEMMEM)
```

Comment:
Copy pasta

Reply:
Oops.

Reply:
I remade the commit and force pushed my branch, rather than polluting history with bad changes.  Hope that is acceptable.

Reply:
Sure, force push is always preferred